File: c/02-count-characters/01.c
# include <stdio.h>
#include <string.h>

int main() {
  char input[250];
  printf("What is the input string? ");
  fgets(input, 250, stdin);
  input[strcspn(input, "\n")] = 0;
  int length = strlen(input);
  printf("%s has %d characters", input, length);
  return 0;
}

File: lua/02-count-characters/01.mjs
import * as readline from 'node:readline/promises';
import { stdin, stdout } from 'node:process';

const rl = readline.createInterface({
  input: stdin,
  output: stdout,
})

const word = await rl.question('What is the input string? ');
rl.close();

const numberOfCharacters = word.length;

console.log(`${word} has ${numberOfCharacters} characters`);
